std/assembly/index.d.ts

```diff
@@ -151,6 +151,8 @@ declare function div<T extends i32 | i64 | f32 | f64>(left: T, right: T): T;
 declare function eq<T extends i32 | i64 | f32 | f64>(left: T, right: T): i32;
 /** Return 0 if two numbers are equal to each other, 1 otherwise. */
 declare function ne<T extends i32 | i64 | f32 | f64>(left: T, right: T): i32;
+/** Computes the rem of two positive integers. */
```

Comment:
```suggestion
/** Computes the remainder of two integers. */
```

Reply:
done in https://github.com/AssemblyScript/assemblyscript/pull/2306/commits/1130f0aac50bf058c79d91228484c4e872b53771